wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ViewModel/ConversationListViewModel.swift

```diff
@@ -232,6 +234,17 @@ final class ConversationListViewModel: NSObject {
 
     private func setupObservers() {
         conversationDirectoryToken = userSession?.conversationDirectory.addObserver(self)
+
+        guard let user = userSession?.selfUser as? ZMUser else { return }
+
+        user.publisher(for: \.teamIdentifier)
+            .receive(on: RunLoop.main)
+            .removeDuplicates()
```

Comment:
```suggestion
            .removeDuplicates()
            .receive(on: RunLoop.main)
```
can we swap the operators ?

Reply:
Good idea. Fixed!